torchdata/dataloader2/graph/utils.py

```diff
@@ -34,6 +30,33 @@ def helper(g) -> None:  # pyre-ignore
     return dps
 
 
+def list_dps(graph: DataPipeGraph, exclude_dps: Optional[Union[DataPipe, List[DataPipe]]] = None) -> List[DataPipe]:
```

Comment:
If a DataPipe is in `exclude_dps`, its predecessors will also be ignored. It seems like that is intended, if so, can we mention that in the docstring?